cpp/include/experimental/graph_view.hpp

```diff
@@ -321,6 +333,10 @@ class graph_view_t<vertex_t,
 
   bool is_weighted() const { return adj_matrix_partition_weights_.size() > 0; }
 
+  partition_t<vertex_t> get_partition() const {
+    return partition_;
+  }
+
```

Comment:
Better delete this, this is intended to be internal and better keep in that way. You can access vertex partition offsets using get_vertex_partition_first(size_t vertex_partition_idx) & get_vertex_partition_last(size_t vertex_partition_idx)

Reply:
See above comment.